ccl/backupccl: TestScheduleBackup failed

[(ccl/backupccl).TestScheduleBackup failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2174981&tab=buildLog) on [master@be8c1a17b940ecc56dc3521ab8264f662c88072c](https://github.com/cockroachdb/cockroach/commits/be8c1a17b940ecc56dc3521ab8264f662c88072c):

```
I200812 16:47:01.216927 326385 server/config.go:626  [n?] Pebble cache size: 128 MiB
I200812 16:47:01.216939 326385 server/config.go:626  [n?] store 0: in-memory, size 0 B
I200812 16:47:01.245178 326385 server/init.go:173  [n?] no stores bootstrapped and --join flag specified, awaiting init command or join with an already initialized node.
I200812 16:47:01.245286 326385 server/init.go:193  [n?] **** cluster b0f30352-d0c4-401e-bbf3-80d598292f6c has been created
I200812 16:47:01.246007 326851 server/server.go:1394  [n?] connecting to gossip network to verify cluster ID "b0f30352-d0c4-401e-bbf3-80d598292f6c"
I200812 16:47:01.246108 326385 gossip/gossip.go:403  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:44457" > attrs:<> locality:<> ServerVersion:<major_val:20 minor_val:1 patch:0 unstable:15 > build_tag:"v20.2.0-alpha.1-2066-gbe8c1a17b9" started_at:1597250821246095972 cluster_name:"" sql_address:<network_field:"tcp" address_field:"127.0.0.1:43071" > tenant_address:<network_field:"tcp" address_field:"127.0.0.1:36139" > 
W200812 16:47:01.252579 326926 kv/kvserver/store.go:1665  [n1,s1,r6/1:/Table/{SystemCon…-11}] could not gossip system config: [NotLeaseHolderError] r6: replica (n1,s1):1 not lease holder; lease holder unknown
I200812 16:47:01.254037 326851 server/server.go:1397  [n1] node connected via gossip
I200812 16:47:01.257207 326385 server/node.go:413  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=19 KiB), ranges=35, leases=2, queries=0.00, writes=0.00, bytesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=166.00 pMax=16466.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I200812 16:47:01.257381 326385 kv/kvserver/stores.go:236  [n1] read 0 node addresses from persistent storage
I200812 16:47:01.257454 326385 server/node.go:469  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I200812 16:47:01.258248 326385 server/server.go:1506  [n1] starting https server at 127.0.0.1:39531 (use: 127.0.0.1:39531)
I200812 16:47:01.263747 326385 server/server.go:1511  [n1] starting postgres server at 127.0.0.1:43071 (use: 127.0.0.1:43071)
I200812 16:47:01.263809 326385 server/server.go:1513  [n1] starting grpc server at 127.0.0.1:44457
I200812 16:47:01.263832 326385 server/server.go:1514  [n1] advertising CockroachDB node at 127.0.0.1:44457
I200812 16:47:01.280818 326863 kv/kvserver/queue.go:582  [n1,s1,r8/1:/Table/1{2-3}] rate limited in MaybeAdd (merge): throttled on async limiting semaphore
I200812 16:47:01.272623 326385 sql/sqlliveness/slinstance/slinstance.go:251  [n1] starting SQL liveness instance
I200812 16:47:01.282592 326963 sql/event_log.go:143  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:127.0.0.1:44457 Attrs: Locality: ServerVersion:20.1-15 BuildTag:v20.2.0-alpha.1-2066-gbe8c1a17b9 StartedAt:1597250821246095972 LocalityAddress:[] ClusterName: SQLAddress:127.0.0.1:43071 TenantAddress:127.0.0.1:36139} ClusterID:b0f30352-d0c4-401e-bbf3-80d598292f6c StartedAt:1597250821246095972 LastUp:1597250821246095972}
I200812 16:47:01.283231 326966 sql/temporary_schema.go:487  [n1] running temporary object cleanup background job
I200812 16:47:01.284196 326966 sql/temporary_schema.go:522  [n1] found 0 temporary schemas
I200812 16:47:01.284562 326966 sql/temporary_schema.go:525  [n1] early exiting temporary schema cleaner as no temporary schemas were found
I200812 16:47:01.284927 326966 sql/temporary_schema.go:526  [n1] completed temporary object cleanup job
I200812 16:47:01.285305 326966 sql/temporary_schema.go:609  [n1] temporary object cleaner next scheduled to run at 2020-08-12 17:17:01.283115027 +0000 UTC
I200812 16:47:01.286436 327071 kv/kvserver/replica_rangefeed.go:602  [n1,s1,r6/1:/Table/{SystemCon…-11}] RangeFeed closed timestamp is empty
I200812 16:47:01.287225 326965 sql/sqlliveness/slstorage/slstorage.go:273  [n1] inserted sqlliveness session a25d1536b657445b82ba92ae75511c74
I200812 16:47:01.287622 326965 sql/sqlliveness/slinstance/slinstance.go:144  [n1] Created new SQL liveness session a25d1536b657445b82ba92ae75511c74
I200812 16:47:01.288756 326964 sql/sqlliveness/slstorage/slstorage.go:252  [n1] Deleted 0 expired SQL liveness sessions
I200812 16:47:01.297394 327095 sql/event_log.go:143  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I200812 16:47:01.320043 327102 sql/event_log.go:143  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:20.1-15 User:root}
I200812 16:47:01.335569 326962 server/server_update.go:55  [n1] no need to upgrade, cluster already at the newest version
I200812 16:47:01.339015 327201 sql/event_log.go:143  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:7f7e0140-251f-4ef3-9b94-e8e1eb1055b0 User:root}
I200812 16:47:01.346368 327430 sql/event_log.go:143  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I200812 16:47:01.350899 327547 sql/event_log.go:143  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I200812 16:47:01.372276 326385 server/server_sql.go:723  [n1] done ensuring all necessary migrations have run
I200812 16:47:01.372356 326385 server/server.go:1888  [n1] serving sql connections
I200812 16:47:01.373445 327768 sqlmigrations/migrations.go:735  [n1] starting wait for upgrade finalization before schema change job migration
I200812 16:47:01.373684 327768 sqlmigrations/migrations.go:767  [n1] starting schema change job migration
I200812 16:47:01.389672 327736 sql/event_log.go:143  [n1,client=127.0.0.1:60906,hostssl,user=root] Event: "create_database", target: 52, info: {DatabaseName:db Statement:CREATE DATABASE db User:root}
I200812 16:47:01.390655 327768 sqlmigrations/migrations.go:1185  [schema-change-job-migration] evaluating tables for creating jobs
I200812 16:47:01.390703 327768 sqlmigrations/migrations.go:787  [n1] schema change job migration completed
I200812 16:47:01.410198 327736 sql/event_log.go:143  [n1,client=127.0.0.1:60906,hostssl,user=root] Event: "create_table", target: 53, info: {TableName:db.public.t1 Statement:CREATE TABLE t1 (a INT8) User:root}
I200812 16:47:01.411513 327760 kv/kvserver/replica_command.go:414  [n1,split,s1,r35/1:/{Table/39-Max}] initiating a split of this range at key /Table/53 [r36] (zone config)
I200812 16:47:01.425956 327736 sql/event_log.go:143  [n1,client=127.0.0.1:60906,hostssl,user=root] Event: "create_table", target: 54, info: {TableName:db.public.t2 Statement:CREATE TABLE t2 (b INT8) User:root}
I200812 16:47:01.426829 327779 kv/kvserver/replica_command.go:414  [n1,split,s1,r36/1:/{Table/53-Max}] initiating a split of this range at key /Table/54 [r37] (zone config)
I200812 16:47:01.447735 327736 sql/event_log.go:143  [n1,client=127.0.0.1:60906,hostssl,user=root] Event: "create_table", target: 55, info: {TableName:db.public.t3 Statement:CREATE TABLE t3 (c INT8) User:root}
I200812 16:47:01.449779 328130 kv/kvserver/replica_command.go:414  [n1,split,s1,r37/1:/{Table/54-Max}] initiating a split of this range at key /Table/55 [r38] (zone config)
I200812 16:47:01.471617 327736 sql/event_log.go:143  [n1,client=127.0.0.1:60906,hostssl,user=root] Event: "create_database", target: 56, info: {DatabaseName:other_db Statement:CREATE DATABASE other_db User:root}
I200812 16:47:01.499533 327736 sql/event_log.go:143  [n1,client=127.0.0.1:60906,hostssl,user=root] Event: "create_table", target: 57, info: {TableName:other_db.public.t1 Statement:CREATE TABLE t1 (a INT8) User:root}
I200812 16:47:01.501204 327726 kv/kvserver/replica_command.go:414  [n1,split,s1,r38/1:/{Table/55-Max}] initiating a split of this range at key /Table/57 [r39] (zone config)
--- FAIL: TestScheduleBackup (1.01s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestScheduleBackup PKG=./pkg/ccl/backupccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestScheduleBackup.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/backupccl).TestScheduleBackup failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2175227&tab=buildLog) on [master@89208ec8b5350fd34036b559a258403e820deda6](https://github.com/cockroachdb/cockroach/commits/89208ec8b5350fd34036b559a258403e820deda6):

```
I200812 17:34:28.227349 332581 server/config.go:623  [n?] 1 storage engine initialized
I200812 17:34:28.227535 332581 server/config.go:626  [n?] Pebble cache size: 128 MiB
I200812 17:34:28.227653 332581 server/config.go:626  [n?] store 0: in-memory, size 0 B
I200812 17:34:28.241295 332581 server/init.go:173  [n?] no stores bootstrapped and --join flag specified, awaiting init command or join with an already initialized node.
I200812 17:34:28.246138 332581 server/init.go:193  [n?] **** cluster 116059c6-c0b8-476d-aafb-a102ac98d211 has been created
I200812 17:34:28.246507 335654 server/server.go:1394  [n?] connecting to gossip network to verify cluster ID "116059c6-c0b8-476d-aafb-a102ac98d211"
I200812 17:34:28.246722 332581 gossip/gossip.go:403  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:33741" > attrs:<> locality:<> ServerVersion:<major_val:20 minor_val:1 patch:0 unstable:15 > build_tag:"v20.2.0-alpha.1-2071-g89208ec8b5" started_at:1597253668246710389 cluster_name:"" sql_address:<network_field:"tcp" address_field:"127.0.0.1:39509" > tenant_address:<network_field:"tcp" address_field:"127.0.0.1:43461" > 
I200812 17:34:28.254558 335654 server/server.go:1397  [n1] node connected via gossip
W200812 17:34:28.256332 335729 kv/kvserver/store.go:1665  [n1,s1,r6/1:/Table/{SystemCon…-11}] could not gossip system config: [NotLeaseHolderError] r6: replica (n1,s1):1 not lease holder; lease holder unknown
I200812 17:34:28.257765 332581 server/node.go:413  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=19 KiB), ranges=35, leases=2, queries=0.00, writes=0.00, bytesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=166.00 pMax=16466.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I200812 17:34:28.258028 332581 kv/kvserver/stores.go:236  [n1] read 0 node addresses from persistent storage
I200812 17:34:28.258552 332581 server/node.go:469  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I200812 17:34:28.259251 332581 server/server.go:1506  [n1] starting https server at 127.0.0.1:34179 (use: 127.0.0.1:34179)
I200812 17:34:28.264474 332581 server/server.go:1511  [n1] starting postgres server at 127.0.0.1:39509 (use: 127.0.0.1:39509)
I200812 17:34:28.264666 332581 server/server.go:1513  [n1] starting grpc server at 127.0.0.1:33741
I200812 17:34:28.265117 332581 server/server.go:1514  [n1] advertising CockroachDB node at 127.0.0.1:33741
I200812 17:34:28.310346 332581 sql/sqlliveness/slinstance/slinstance.go:251  [n1] starting SQL liveness instance
I200812 17:34:28.313161 335880 kv/kvserver/replica_rangefeed.go:602  [n1,s1,r6/1:/Table/{SystemCon…-11}] RangeFeed closed timestamp is empty
I200812 17:34:28.315376 335811 sql/temporary_schema.go:487  [n1] running temporary object cleanup background job
I200812 17:34:28.316520 335811 sql/temporary_schema.go:522  [n1] found 0 temporary schemas
I200812 17:34:28.316727 335811 sql/temporary_schema.go:525  [n1] early exiting temporary schema cleaner as no temporary schemas were found
I200812 17:34:28.317305 335811 sql/temporary_schema.go:526  [n1] completed temporary object cleanup job
I200812 17:34:28.317678 335811 sql/temporary_schema.go:609  [n1] temporary object cleaner next scheduled to run at 2020-08-12 18:04:28.313343984 +0000 UTC
I200812 17:34:28.322704 335760 sql/event_log.go:143  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:127.0.0.1:33741 Attrs: Locality: ServerVersion:20.1-15 BuildTag:v20.2.0-alpha.1-2071-g89208ec8b5 StartedAt:1597253668246710389 LocalityAddress:[] ClusterName: SQLAddress:127.0.0.1:39509 TenantAddress:127.0.0.1:43461} ClusterID:116059c6-c0b8-476d-aafb-a102ac98d211 StartedAt:1597253668246710389 LastUp:1597253668246710389}
I200812 17:34:28.323739 335810 sql/sqlliveness/slstorage/slstorage.go:273  [n1] inserted sqlliveness session 37ee30370e864073a96c2e0618e43286
I200812 17:34:28.323797 335810 sql/sqlliveness/slinstance/slinstance.go:144  [n1] Created new SQL liveness session 37ee30370e864073a96c2e0618e43286
I200812 17:34:28.327457 335761 sql/sqlliveness/slstorage/slstorage.go:252  [n1] Deleted 0 expired SQL liveness sessions
I200812 17:34:28.330272 335891 sql/event_log.go:143  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I200812 17:34:28.341420 335915 sql/event_log.go:143  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:20.1-15 User:root}
I200812 17:34:28.375753 335759 server/server_update.go:55  [n1] no need to upgrade, cluster already at the newest version
I200812 17:34:28.380257 336168 sql/event_log.go:143  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:9b31e5ba-e79a-466c-9ee4-60e5d2333e0c User:root}
I200812 17:34:28.391018 336393 sql/event_log.go:143  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I200812 17:34:28.407391 336400 sql/event_log.go:143  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I200812 17:34:28.460139 332581 server/server_sql.go:723  [n1] done ensuring all necessary migrations have run
I200812 17:34:28.460203 332581 server/server.go:1888  [n1] serving sql connections
I200812 17:34:28.461039 336561 sqlmigrations/migrations.go:735  [n1] starting wait for upgrade finalization before schema change job migration
I200812 17:34:28.463396 336561 sqlmigrations/migrations.go:767  [n1] starting schema change job migration
I200812 17:34:28.478125 336565 sql/event_log.go:143  [n1,client=127.0.0.1:52102,hostssl,user=root] Event: "create_database", target: 52, info: {DatabaseName:db Statement:CREATE DATABASE db User:root}
I200812 17:34:28.478181 336561 sqlmigrations/migrations.go:1185  [schema-change-job-migration] evaluating tables for creating jobs
I200812 17:34:28.478904 336561 sqlmigrations/migrations.go:787  [n1] schema change job migration completed
I200812 17:34:28.492422 336565 sql/event_log.go:143  [n1,client=127.0.0.1:52102,hostssl,user=root] Event: "create_table", target: 53, info: {TableName:db.public.t1 Statement:CREATE TABLE t1 (a INT8) User:root}
I200812 17:34:28.502252 336746 kv/kvserver/replica_command.go:414  [n1,split,s1,r35/1:/{Table/39-Max}] initiating a split of this range at key /Table/53 [r36] (zone config)
I200812 17:34:28.516906 336565 sql/event_log.go:143  [n1,client=127.0.0.1:52102,hostssl,user=root] Event: "create_table", target: 54, info: {TableName:db.public.t2 Statement:CREATE TABLE t2 (b INT8) User:root}
I200812 17:34:28.519648 336666 kv/kvserver/replica_command.go:414  [n1,split,s1,r36/1:/{Table/53-Max}] initiating a split of this range at key /Table/54 [r37] (zone config)
I200812 17:34:28.586969 336565 sql/event_log.go:143  [n1,client=127.0.0.1:52102,hostssl,user=root] Event: "create_table", target: 55, info: {TableName:db.public.t3 Statement:CREATE TABLE t3 (c INT8) User:root}
I200812 17:34:28.588507 336946 kv/kvserver/replica_command.go:414  [n1,split,s1,r37/1:/{Table/54-Max}] initiating a split of this range at key /Table/55 [r38] (zone config)
I200812 17:34:28.609710 336565 sql/event_log.go:143  [n1,client=127.0.0.1:52102,hostssl,user=root] Event: "create_database", target: 56, info: {DatabaseName:other_db Statement:CREATE DATABASE other_db User:root}
I200812 17:34:28.621607 336565 sql/event_log.go:143  [n1,client=127.0.0.1:52102,hostssl,user=root] Event: "create_table", target: 57, info: {TableName:other_db.public.t1 Statement:CREATE TABLE t1 (a INT8) User:root}
I200812 17:34:28.623606 337105 kv/kvserver/replica_command.go:414  [n1,split,s1,r38/1:/{Table/55-Max}] initiating a split of this range at key /Table/57 [r39] (zone config)
--- FAIL: TestScheduleBackup (1.16s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestScheduleBackup PKG=./pkg/ccl/backupccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestScheduleBackup.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/backupccl).TestScheduleBackup failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2177109&tab=buildLog) on [master@457f1af5d19e0bcef9b5ea9e9125b3332717a86c](https://github.com/cockroachdb/cockroach/commits/457f1af5d19e0bcef9b5ea9e9125b3332717a86c):

```
I200813 05:33:04.323416 313688 server/server.go:815  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I200813 05:33:04.323795 313688 server/config.go:623  [n?] 1 storage engine initialized
I200813 05:33:04.323830 313688 server/config.go:626  [n?] Pebble cache size: 128 MiB
I200813 05:33:04.323841 313688 server/config.go:626  [n?] store 0: in-memory, size 0 B
I200813 05:33:04.326727 313688 server/init.go:173  [n?] no stores bootstrapped and --join flag specified, awaiting init command or join with an already initialized node.
I200813 05:33:04.326769 313688 server/init.go:193  [n?] **** cluster c5544f19-e153-4bc1-9a0f-0dacb904f7f7 has been created
I200813 05:33:04.326820 313688 gossip/gossip.go:403  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:42105" > attrs:<> locality:<> ServerVersion:<major_val:20 minor_val:1 patch:0 unstable:15 > build_tag:"v20.2.0-alpha.1-2102-g457f1af5d1-dirty" started_at:1597296784326806987 cluster_name:"" sql_address:<network_field:"tcp" address_field:"127.0.0.1:33929" > tenant_address:<network_field:"tcp" address_field:"127.0.0.1:45525" > 
I200813 05:33:04.326916 313769 server/server.go:1394  [n1] connecting to gossip network to verify cluster ID "c5544f19-e153-4bc1-9a0f-0dacb904f7f7"
I200813 05:33:04.329286 313688 server/node.go:413  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=19 KiB), ranges=35, leases=0, queries=0.00, writes=0.00, bytesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=166.00 pMax=16530.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
W200813 05:33:04.329385 313844 kv/kvserver/store.go:1665  [n1,s1,r6/1:/Table/{SystemCon…-11}] could not gossip system config: [NotLeaseHolderError] r6: replica (n1,s1):1 not lease holder; lease holder unknown
I200813 05:33:04.329419 313688 kv/kvserver/stores.go:236  [n1] read 0 node addresses from persistent storage
I200813 05:33:04.329500 313769 server/server.go:1397  [n1] node connected via gossip
I200813 05:33:04.329510 313688 server/node.go:469  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I200813 05:33:04.329599 313688 server/server.go:1506  [n1] starting https server at 127.0.0.1:38871 (use: 127.0.0.1:38871)
I200813 05:33:04.329633 313688 server/server.go:1511  [n1] starting postgres server at 127.0.0.1:33929 (use: 127.0.0.1:33929)
I200813 05:33:04.329657 313688 server/server.go:1513  [n1] starting grpc server at 127.0.0.1:42105
I200813 05:33:04.329672 313688 server/server.go:1514  [n1] advertising CockroachDB node at 127.0.0.1:42105
I200813 05:33:04.335510 313688 sql/sqlliveness/slinstance/slinstance.go:251  [n1] starting SQL liveness instance
I200813 05:33:04.346467 313889 sql/temporary_schema.go:487  [n1] running temporary object cleanup background job
I200813 05:33:04.347405 313889 sql/temporary_schema.go:522  [n1] found 0 temporary schemas
I200813 05:33:04.347438 313889 sql/temporary_schema.go:525  [n1] early exiting temporary schema cleaner as no temporary schemas were found
I200813 05:33:04.347465 313889 sql/temporary_schema.go:526  [n1] completed temporary object cleanup job
I200813 05:33:04.347481 313889 sql/temporary_schema.go:609  [n1] temporary object cleaner next scheduled to run at 2020-08-13 06:03:04.344373496 +0000 UTC
I200813 05:33:04.356914 313582 kv/kvserver/replica_rangefeed.go:602  [n1,s1,r6/1:/Table/{SystemCon…-11}] RangeFeed closed timestamp is empty
I200813 05:33:04.357324 313888 sql/sqlliveness/slstorage/slstorage.go:275  [n1] inserted sqlliveness session 7073566c20be4899847e49f2d3c2eef7
I200813 05:33:04.357371 313888 sql/sqlliveness/slinstance/slinstance.go:144  [n1] Created new SQL liveness session 7073566c20be4899847e49f2d3c2eef7
I200813 05:33:04.357593 313886 sql/event_log.go:143  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:127.0.0.1:42105 Attrs: Locality: ServerVersion:20.1-15 BuildTag:v20.2.0-alpha.1-2102-g457f1af5d1-dirty StartedAt:1597296784326806987 LocalityAddress:[] ClusterName: SQLAddress:127.0.0.1:33929 TenantAddress:127.0.0.1:45525} ClusterID:c5544f19-e153-4bc1-9a0f-0dacb904f7f7 StartedAt:1597296784326806987 LastUp:1597296784326806987}
I200813 05:33:04.359569 313924 sql/event_log.go:143  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I200813 05:33:04.372580 314098 sql/event_log.go:143  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:20.1-15 User:root}
I200813 05:33:04.383987 314286 sql/event_log.go:143  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:5bb3bf08-aa55-4e06-8a26-9de290e36181 User:root}
I200813 05:33:04.389251 314498 sql/event_log.go:143  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I200813 05:33:04.392221 314505 sql/event_log.go:143  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I200813 05:33:04.404204 313885 server/server_update.go:55  [n1] no need to upgrade, cluster already at the newest version
I200813 05:33:04.408633 313688 server/server_sql.go:723  [n1] done ensuring all necessary migrations have run
I200813 05:33:04.408681 313688 server/server.go:1888  [n1] serving sql connections
I200813 05:33:04.408760 314676 sqlmigrations/migrations.go:735  [n1] starting wait for upgrade finalization before schema change job migration
I200813 05:33:04.408970 314676 sqlmigrations/migrations.go:767  [n1] starting schema change job migration
I200813 05:33:04.416017 314676 sqlmigrations/migrations.go:1185  [schema-change-job-migration] evaluating tables for creating jobs
I200813 05:33:04.416043 314676 sqlmigrations/migrations.go:787  [n1] schema change job migration completed
I200813 05:33:04.421478 314655 sql/event_log.go:143  [n1,client=127.0.0.1:37244,hostssl,user=root] Event: "create_database", target: 52, info: {DatabaseName:db Statement:CREATE DATABASE db User:root}
I200813 05:33:04.431055 314655 sql/event_log.go:143  [n1,client=127.0.0.1:37244,hostssl,user=root] Event: "create_table", target: 53, info: {TableName:db.public.t1 Statement:CREATE TABLE t1 (a INT8) User:root}
I200813 05:33:04.432081 314744 kv/kvserver/replica_command.go:414  [n1,split,s1,r35/1:/{Table/39-Max}] initiating a split of this range at key /Table/53 [r36] (zone config)
I200813 05:33:04.444549 314655 sql/event_log.go:143  [n1,client=127.0.0.1:37244,hostssl,user=root] Event: "create_table", target: 54, info: {TableName:db.public.t2 Statement:CREATE TABLE t2 (b INT8) User:root}
I200813 05:33:04.445292 314892 kv/kvserver/replica_command.go:414  [n1,split,s1,r36/1:/{Table/53-Max}] initiating a split of this range at key /Table/54 [r37] (zone config)
I200813 05:33:04.455907 314655 sql/event_log.go:143  [n1,client=127.0.0.1:37244,hostssl,user=root] Event: "create_table", target: 55, info: {TableName:db.public.t3 Statement:CREATE TABLE t3 (c INT8) User:root}
I200813 05:33:04.456819 315055 kv/kvserver/replica_command.go:414  [n1,split,s1,r37/1:/{Table/54-Max}] initiating a split of this range at key /Table/55 [r38] (zone config)
I200813 05:33:04.473063 314655 sql/event_log.go:143  [n1,client=127.0.0.1:37244,hostssl,user=root] Event: "create_database", target: 56, info: {DatabaseName:other_db Statement:CREATE DATABASE other_db User:root}
I200813 05:33:04.481945 314655 sql/event_log.go:143  [n1,client=127.0.0.1:37244,hostssl,user=root] Event: "create_table", target: 57, info: {TableName:other_db.public.t1 Statement:CREATE TABLE t1 (a INT8) User:root}
I200813 05:33:04.482813 315118 kv/kvserver/replica_command.go:414  [n1,split,s1,r38/1:/{Table/55-Max}] initiating a split of this range at key /Table/57 [r39] (zone config)
--- FAIL: TestScheduleBackup (0.47s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestScheduleBackup PKG=./pkg/ccl/backupccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestScheduleBackup.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/ccl/backupccl/create_scheduled_backup_test.go
+++ b/pkg/ccl/backupccl/create_scheduled_backup_test.go
@@ -481,6 +481,7 @@ INSERT INTO t1 values (-1), (10), (-100);
 			schedules, err := th.createBackupSchedule(t, tc.schedule, destination)
 			require.NoError(t, err)
 			require.EqualValues(t, 1, len(schedules))
+			defer th.sqlDB.Exec(t, "DROP SCHEDULE $1", schedules[0].ScheduleID())
 
 			// Force the schedule to execute.
 			th.env.SetTime(schedules[0].NextRun().Add(time.Second))

